app/src/main/java/com/google/samples/apps/sunflower/utilities/Screen.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples.apps.sunflower.utilities
+
+import androidx.navigation.NamedNavArgument
+import androidx.navigation.NavType
+import androidx.navigation.navArgument
+
+sealed class Screen(
```

Comment:
Rather than putting this under the `utilities` package, I recommend adding this in `compose` package.

Reply:
Thanks, I've addressed the feedback in commit 'Moved Screen class to compose package'. Could you take a look, @arriolac ?

[Commit Link](https://github.com/sahilsk3333/sunflower/commit/e6a9ea1eaa4a23d1fa76eb7d4179a09670dfce7b)